packages/turf-clusters-dbscan/index.ts

```diff
@@ -66,6 +66,9 @@ function clustersDbscan(
   } = {}
 ): FeatureCollection<Point, DbscanProps> {
   // Input validation being handled by Typescript
+  // TODO oops! No it isn't. Typescript doesn't do runtime checking. We should
+  // re-enable these checks, though will have to wait for a major version bump
+  // as more restrictive checks could break currently working code.
```

Comment:
Yeah there's a philosophical argument on whether these should be static checks or runtime checks. We've gone back and forth on it over the years.